viewer/src/datamodels/cad/rendering/EffectRenderManager.ts

```diff
@@ -11,34 +11,39 @@ import { CadNode } from '..';
 import { Cognite3DModel } from '@/migration';
 import { RootSectorNode } from '../sector/RootSectorNode';
 
+type Object3DStructure = {
```

Comment:
This class is starting to get a little messy, and I think it is ripe for refactoring everything that has to do with scene building into its own component / class, and let the render manager deal with rendering and and render targets. Not something we necessarily have to do right now, but I think we should at the very least track it under tech debt.

Reply:
Refactored this, have another looksie and see what you think